### DISCUSSION TITLE
Error in regex rules validation using OR ( | ) operator

### DISCUSSION THREAD

**Discussion by rsimonru** [livewire/livewire#4053] [1. Help] [answered]
Hello.

I'm trying to validade a regex rule that use the OR ("|") operator and get an error because the regex OR is the same than rules separator.

¿Is there a way to use this operator in a regex rule?

Example rule:  
'value' => "required|regex:/^\s*-?((\d{1,3}(\.(\d){3})*)|(\d*))(,\d{1,2})?\s?(\€)?\s*$/",

**Comment by joshhanley** [comment, marked as answer]
@rsimonru I think that is more of a Laravel issue, not a Livewire one.

But you can just split the rules out into an array and that should work.

```php
'value' => ["required", "regex:/^\s*-?((\d{1,3}(.(\d){3}))|(\d))(,\d{1,2})?\s?(\€)?\s*$/"],
```

Hope this helps!

**Reply by rsimonru** [reply]
@joshhanley, you are right. Using an array solve the problem.

Thanks!

**Reply by joshhanley** [reply]
No worries! 🙂
